src/main/java/org/openjdk/jextract/impl/UnsupportedFilter.java

```diff
@@ -70,6 +70,8 @@ public Declaration.Scoped scan(Declaration.Scoped header) {
 
     @Override
     public Void visitFunction(Function funcTree, Declaration firstNamedParent) {
+        Utils.forEachNested(funcTree, s -> s.accept(this, firstNamedParent));
```

Comment:
Note: this method was not recursing into declarations nested in the return/parameters - this caused issues when a nested declaration happened to contain unsupported types, as `OutputFactory` would attempt to generate something for which the layout string cannot be generated.